apps/server/lib/lexical/server/state.ex

```diff
@@ -170,17 +170,21 @@ defmodule Lexical.Server.State do
     end
   end
 
-  def apply(%__MODULE__{} = state, %DidOpen{lsp: event}) do
-    %TextDocument.Item{text: text, uri: uri, version: version} =
-      text_document = event.text_document
-
-    case Document.Store.open(uri, text, version) do
+  def apply(%__MODULE__{} = state, %DidOpen{} = did_open) do
+    %TextDocument.Item{
+      text: text,
+      uri: uri,
+      version: version,
+      language_id: language_id
+    } = did_open.lsp.text_document
```

Comment:
do we need the lsp version? I think this will work with `did_open.text_document`

Reply:
I see what you mean, done.